common.blocks/form-field/_type/form-field_type_hidden.browser.js

```diff
@@ -0,0 +1,18 @@
+modules.define('form-field', function(provide, FormField) {
+
+    provide(FormField.decl({ modName : 'type', modVal : 'hidden' }, {
+
+        getName : function() {
+            return this.params.name;
+        },
+
+        setVal : function(val) {
+            this.params.value = val;
```

Comment:
> By DOM you mean that the block should render an <input type="hidden" value="..." name="..."> ? I thought about that, but what difference does it make?

Agree that if browser does have some js we don't need this in DOM. But still we should note that in docs somehow ;-(

> Also, what you mean by values on server side?

I mean simple form submit if browser doesn't have `js`. It's a graceful-degradation issue for bemhtml/bh.